delete.js

```diff
@@ -38,5 +45,6 @@ const deletionUi = {
       }
     });
   }
-};
-deletionUi.initiaize();
+}
+const deleter = new Deleter(addressMain),
+deletionUi = new DeletionUi(deleter, addressMain).initiaize();
```

Comment:
Does `initialize()` return anything? `initialize()` sounds like a command action.
Why are you saving the result of calling `initialize()` in `deletioUi` ?
are you going to do something with `deletionUi`?

Also, have you checked that the syntax `new DeletionUi(deleter, addressMain).initiaize();` is right? You need to wrap the instantiation with a parenthesis.


I think this is what you want to do:
```js
(new DeletionUi(deleter, addressMain)).initialize();
```

Reply:
now I understand